testsuite/features/secondary/proxy_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) 2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@proxy
+@private_net
+@pxeboot_minion
+@scope_cobbler
+Feature: PXE boot a terminal with Cobbler
+  In order to automate client system installations in SUSE Manager
+  As the system administrator
+  I want to PXE boot one host with Cobbler
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Configure PXE part of DHCP on the proxy
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Formulas" in the content area
+    And I follow first "Dhcpd" in the content area
+    And I enter the local IP address of "proxy" in pxeboot next server field
+    And I enter "pxelinux.0" in pxeboot filename field
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
+
+  Scenario: Configure PXE part of DNS on the proxy
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Formulas" in the content area
+    And I follow first "Bind" in the content areaoto
+    # general information:
+    #   (Avahi does not cross networks, so we need to cheat by serving tf.local)
+    And I press "Add Item" in configured zones section
+    And I enter "tf.local" in third configured zone name field
+    # direct zone tf.local:
+    #   (Avahi does not cross networks, so we need to cheat by serving tf.local)
+    And I scroll to the top of the page
+    And I press "Add Item" in available zones section
+    And I enter "tf.local" in third available zone name field
+    And I enter "master/db.tf.local" in third file name field
+    And I enter the hostname of "proxy" in third name server field
+    And I enter "admin@tf.local." in third contact field
+    And I press "Add Item" in third A section
+    And I enter the hostname of "proxy" in fifth A name field
+    And I enter the IP address of "proxy" in fifth A address field
+    And I press "Add Item" in third A section
+    And I enter the hostname of "server" in sixth A name field
+    And I enter the IP address of "server" in sixth A address field
+    And I press "Add Item" in third NS section
+    And I enter the hostname of "proxy" in third NS field
+    # end
+    And I scroll to the top of the page
+    And I should see a "Bind" text
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
+
+  Scenario: Apply the highstate after the formula setup
+    Given I am on the Systems overview page of this "proxy"
```

Comment:
I would say you don't need the first step, and it will reduce some navigation steps.
This first step contains too many steps on it, we don't need because the previous scenario ends in a state where your browser still inside the proxy page.

Reply:
You are right.  Good point!